sdk/src/tx/retryTxSender.ts

```diff
@@ -13,7 +14,7 @@ type ResolveReference = {
 
 export class RetryTxSender extends BaseTxSender {
 	connection: Connection;
-	wallet: IWallet;
+	wallet: Wallet;
```

Comment:
I don't actually necessary wanna stick with this change - see other comment I added below